<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App COVID</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCustomSlideMenu" name="left_menu_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/iconos/iconos covid/coronavirus_covid_corona_virus_bacterium_covid_icon_149135.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>COVID-19</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="but_datos">
            <property name="text">
             <string>Datos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_graficos">
            <property name="text">
             <string>Gráficos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_sesion">
            <property name="text">
             <string>Cerrar sesión</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_salir">
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Ionut Artene</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>IES El Just 2022</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="but_menu">
               <property name="font">
                <font>
                 <family>Syamala Ramana</family>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/iconos/iconos covid/Menu_icon_icon-icons.com_71858.svg</normaloff>
                 <normalon>:/iconos/iconos covid/left-arrow.png</normalon>
                 <disabledoff>:/iconos/iconos covid/Menu_icon_icon-icons.com_71858.svg</disabledoff>:/iconos/iconos covid/Menu_icon_icon-icons.com_71858.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string> MENU</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>COVID-19</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QStackedWidget" name="ventanas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_datos">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QWidget" name="widget" native="true">
                <widget class="QComboBox" name="combo_municipios">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>140</y>
                   <width>171</width>
                   <height>25</height>
                  </rect>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <property name="currentText">
                  <string notr="true"/>
                 </property>
                 <property name="maxVisibleItems">
                  <number>5</number>
                 </property>
                 <property name="placeholderText">
                  <string>Elija un Municipio</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="valencia">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>70</y>
                   <width>112</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Valencia</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="castellon">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>40</y>
                   <width>112</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Castellon</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="alicante">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>100</y>
                   <width>112</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Alicante</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_7">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>50</y>
                   <width>67</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Provincia</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>150</y>
                   <width>81</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Municipios</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QLabel" name="label_10">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>50</y>
                   <width>251</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Incidència acumulada PCR+ 14 días :</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_11">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>80</y>
                   <width>141</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Casos PCR+ 14 días:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_12">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>110</y>
                   <width>67</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Muertes:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_pcr">
                 <property name="geometry">
                  <rect>
                   <x>340</x>
                   <y>50</y>
                   <width>101</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
                <widget class="QLabel" name="label_casos">
                 <property name="geometry">
                  <rect>
                   <x>340</x>
                   <y>80</y>
                   <width>101</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
                <widget class="QLabel" name="label_muertes">
                 <property name="geometry">
                  <rect>
                   <x>340</x>
                   <y>110</y>
                   <width>111</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_graficos">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item alignment="Qt::AlignTop">
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>Grafico COVID</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="gridFrame">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <layout class="QGridLayout" name="line_chart"/>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomSlideMenu</class>
   <extends>QWidget</extends>
   <header location="global">Custom_Widgets.Widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
